docs/.vuepress/config.js

```diff
@@ -72,20 +72,47 @@ module.exports = {
             text: 'Config Reference',
             link: '/config/'
           },
-          {
-            text: 'Plugin Dev Guide',
-            items: [
-              { text: 'Plugin Dev Guide', link: '/dev-guide/plugin-dev.md' },
-              { text: 'UI Plugin Info', link: '/dev-guide/ui-info.md' },
-              { text: 'UI Plugin API', link: '/dev-guide/ui-api.md' },
-              { text: 'UI Localization', link: '/dev-guide/ui-localization.md' }
-            ]
-          },
           {
             text: 'Plugins',
             items: [
-              { text: 'Core plugins', link: '/core-plugins/' },
-              { text: 'Browse plugins', link: 'https://awesomejs.dev/for/vue-cli/' }
+              {
+                text: 'Configuration',
+                items:
+                [
+                  { text: 'Core', link: '/core-plugins/' },
+                ]
+              },
+              {
+                text: 'Discover',
```

Comment:
Although this is using a verb-like label when compared to the rest of the sections here, I would consider this ok as-is for now.